acq4/devices/Scientifica/scientifica.py

```diff
@@ -24,20 +29,21 @@ class Scientifica(Stage):
     The optional 'baudrate' parameter is used to set the baudrate of the device.
     Both valid rates will be attempted when initially connecting.
     """
+
     def __init__(self, man, config, name):
-        # can specify 
-        port = config.pop('port', None)
-        name = config.pop('name', None)
+        # can specify
+        port = config.pop("port", None)
+        name = config.pop("name", None)
```

Comment:
sorry; I must have hit the "autoformat file" button when I was fixing a line, instead of just "autoformat line".